sample/fizzBuzz/fizzbuzz.py

```diff
@@ -1,6 +1,8 @@
 for i in range(1, 101):
-    if i % 3 == 0:
-        print('Fiz')
+    if i % 3 == 0 and i % 5 == 0:
+        print('FizzBuzz')
+    elif i % 3 == 0:
```

Comment:
レビュワーからのコメントがある場合はこのようにコメントができます。

具体的にこうするといいと思うというコードを提示したりもできます。
```suggestion
    if i ** 2 % 3 == 0:
```
（無駄にフェルマーの小定理を使ってみた）